--- src/Codeception/Module/Symfony/NotifierAssertionsTrait.php
+++ src/Codeception/Module/Symfony/NotifierAssertionsTrait.php
@@ -245,8 +245,7 @@

     protected function getNotificationEvents(): NotificationEvents
     {
-        // @phpstan-ignore if.alwaysFalse
-        if (version_compare(Kernel::VERSION, '6.2', '<')) {
+        if (!class_exists(NotificationLoggerListener::class)) {
             Assert::fail('Notifier assertions require Symfony 6.2 or higher.');
         }
